packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2497,15 +2572,32 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
     private submit(
         type: ContainerMessageType,
-        content: any,
+        contents: any,
         localOpMetadata: unknown = undefined,
-        opMetadata: Record<string, unknown> | undefined = undefined,
+        metadata: Record<string, unknown> | undefined = undefined,
     ): void {
         this.verifyNotClosed();
 
         // There should be no ops in detached container state!
         assert(this.attachState !== AttachState.Detached, 0x132 /* "sending ops in detached container" */);
 
+<<<<<<< HEAD
+        const deserializedContent: ContainerRuntimeMessage = { type, contents };
```

Comment:
there are still some unresolved conflicts here

Reply:
I'm still working on resolving these conflicts. The code path that the compression code was originally on no longer exists and so I need to find the right place for it.